bayesapi/resources/column_data.py

```diff
@@ -0,0 +1,27 @@
+import falcon
+
+from bayesapi.resources import BaseResource
+from bayesapi.validation import validate
+
+class ColumnDataResource(BaseResource):
+
+    def on_post(self, req, resp):
+
+        req_vars = validate(self.api_def, 'post', req)
+
+        table_name = self.cfg.table_name
+
+        columns = req_vars['columns']
+
+        with self.bdb.savepoint():
+            query = self.queries.column_data(
+                column_names = ['"{}"'.format(c) for c in columns],
```

Comment:
This can be moved into the query itself, right?

Reply:
I recall that we were going to write a guard for this, but perhaps we just haven't gotten around to it.